src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/DummyGraph.java

```diff
@@ -15,6 +15,12 @@
 
```

Comment:
At some point we need to change the name of this class. This is not dummy but the actual entry point to the RCA framework